<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoyaltyPage</class>
 <widget class="QWidget" name="LoyaltyPage">
  <property name="windowTitle">
   <string>Loyalty</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
            <widget class="QFrame" name="framePol">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayoutPol">
              <property name="spacing">
               <number>6</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayoutPolTitle">
                <item>
                 <widget class="QLabel" name="labelPolTitle">
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>PoL / Loyalty</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacerPolTitle">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="buttonPolRefresh">
                  <property name="text">
                   <string>Refresh</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="labelPolStatus">
                <property name="text">
                 <string>Enter miner payout address and press Refresh.</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayoutPol">
                <property name="horizontalSpacing">
                 <number>12</number>
                </property>
                <property name="verticalSpacing">
                 <number>6</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="labelPolMinerIdText">
                  <property name="text">
                   <string>Miner Address</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" colspan="2">
                 <widget class="QComboBox" name="comboPolMinerAddress">
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::NoInsert</enum>
              </property>
             </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="labelPolTagText">
                  <property name="text">
                   <string>Tag (12B)</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1" colspan="2">
                 <widget class="QLabel" name="labelPolTag">
                  <property name="text">
                   <string notr="true">-</string>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="labelPolBlocksSeenText">
                  <property name="text">
                   <string>Blocks seen</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1" colspan="2">
                 <widget class="QLabel" name="labelPolBlocksSeen">
                  <property name="text">
                   <string notr="true">-</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="labelPolAllowedText">
                  <property name="text">
                   <string>Allowed</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1" colspan="2">
                 <widget class="QLabel" name="labelPolAllowed">
                  <property name="text">
                   <string notr="true">-</string>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="labelPolBaseText">
                  <property name="text">
                   <string>Base</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1" colspan="2">
                 <widget class="QLabel" name="labelPolBase">
                  <property name="text">
                   <string notr="true">-</string>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="labelPolBonusText">
                  <property name="text">
                   <string>Bonus</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1" colspan="2">
                 <widget class="QLabel" name="labelPolBonus">
                  <property name="text">
                   <string notr="true">-</string>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="labelPolLastSeenText">
                  <property name="text">
                   <string>Last seen</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1" colspan="2">
                 <widget class="QLabel" name="labelPolLastSeen">
                  <property name="text">
                   <string notr="true">-</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QLabel" name="labelPolTipText">
                  <property name="text">
                   <string>Tip height</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1" colspan="2">
                 <widget class="QLabel" name="labelPolTipHeight">
                  <property name="text">
                   <string notr="true">-</string>
                  </property>
                 </widget>
                </item>
           
    <item row="8" column="0">
     <widget class="QLabel" name="labelPolPointsText">
      <property name="text">
       <string>Points</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QLabel" name="labelPolPoints">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="labelPolLevelText">
      <property name="text">
       <string>Your Level</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="labelPolLevel">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
            <item row="10" column="1" colspan="2">
             <widget class="QLabel" name="labelPolLevelIcon">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
            </item>
    </layout>
              </item>
             </layout>
            </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
